docker/kafka/Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN git checkout ${AK_BRANCH}
 
 ARG SCALA_VERSION=2.12
 
-RUN ./gradlew -PscalaVersion=${SCALA_VERSION} testJar releaseTarGz
+RUN ./gradlew -PscalaVersion=${SCALA_VERSION} testJar distTar
```

Comment:
I think this one is not required as it's building upstream kafka, not the plugin

Reply:
We will get rid of this file anyway on #262

Reply:
Ah, well, you right, was doing just grep + replace

Reply:
Fixed